core/src/test/scala/unit/kafka/tools/ConsumerPerformanceTest.scala

```diff
@@ -96,6 +97,28 @@ class ConsumerPerformanceTest {
     assertEquals("test", config.topic)
     assertEquals(10, config.numMessages)
   }
+  
+  @Test
+  def testConfigWithRecognizedOptionOverride(): Unit = {
+    val propsFile = TestUtils.tempFile()
+    val propsStream = Files.newOutputStream(propsFile.toPath)
+    propsStream.write("group.id=test_group_id\n".getBytes())
+    propsStream.write("client.id=test_client_id".getBytes())
+    propsStream.close()
```

Comment:
Could you test ```--topic``` to make sure it is not overrided by command-line arguments.

Reply:
`--topic` is a `REQUIRED` argument in command-line, I thank Its value is determined by the command line.

Reply:
So, if config file has the param `topic=test`, It will not take effect.

Reply:
Or could we add ```group.id``` to command-line arguments to make sure it (from file) is NOT override by command-line args?

Reply:
Okey, I finished the local test and submitted the code :)

Reply:
@chia7712  Please take a look :) Is there anything else that needs to be optimized？